specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json

```diff
@@ -0,0 +1,3258 @@
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get server metric configuration for a test": {
+            "$ref": "./examples/GetTestServerMetricsConfig.json"
+          }
+        }
+      }
+    },
+    "/test-runs/{testRunId}": {
+      "delete": {
+        "tags": [
+          "TestRun"
+        ],
+        "summary": "Delete a test run by its name.",
+        "operationId": "TestRun_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/TestRunIdPathParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "The test run is deleted."
+          },
+          "default": {
+            "description": "Load Testing service error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete a test run": {
+            "$ref": "./examples/DeleteTestRun.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "TestRun"
+        ],
+        "summary": "Create and start a new test run with the given name.",
+        "operationId": "TestRun_CreateOrUpdate",
+        "consumes": [
+          "application/merge-patch+json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/TestRunIdPathParameter"
+          },
+          {
+            "in": "query",
+            "name": "oldTestRunId",
+            "description": "Existing test run identifier that should be rerun, if this is provided, the test will run with the JMX file, configuration and app components from the existing test run. You can override the configuration values for new test run in the request body.",
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Load test run model",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TestRun"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The test run is updated.",
+            "schema": {
+              "$ref": "#/definitions/TestRun"
+            }
+          },
```

Comment:
This operation should return a 201 status code in the case where the test run was created.

Reply:
added a 201 response code